capstone/data/datasets.py

```diff
@@ -55,10 +56,13 @@ def __getitem__(self, index: int) -> Tuple[np.ndarray, np.ndarray, np.ndarray]:
 
         if self.structure_required is not None:
             idx = miccai.STRUCTURES.index(self.structure_required)
-            masks = np.expand_dims(masks[idx], axis=0)
-            mask_indicator = np.expand_dims(mask_indicator[idx], axis=0)
+            masks = torch.from_numpy(np.expand_dims(masks[idx], axis=0))
+            mask_indicator = torch.from_numpy(
+                np.expand_dims(mask_indicator[idx], axis=0)
+            )
         else:
-            masks = np.stack(masks)
+            masks = torch.from_numpy(np.stack(masks))
+            mask_indicator = torch.from_numpy(mask_indicator)
 
         return image, masks, mask_indicator
```

Comment:
The `masks` and `mask_indicator` were numpy arrays, and not tensors. Fixed that.

Although this should have no effect on the dataloaders.